plugins/single_plugins/ipc-rules.cpp

```diff
@@ -506,7 +589,38 @@ class ipc_rules_t : public wf::plugin_interface_t, public wf::per_output_tracker
     // Fullscreened rule handler.
     wf::signal::connection_t<wf::view_fullscreen_signal> _fullscreened = [=] (wf::view_fullscreen_signal *ev)
     {
-        send_view_to_subscribes(ev->view, "view-fullscreen");
+        if (clients.empty())
+        {
+            return;
+        }
+
+        nlohmann::json data;
+        data["event"] = "view-fullscreen";
+        data["state"] = ev->state;
```

Comment:
Do we need this since it is available in the data['view']?

Reply:
Probably not, I probably added it while mindlessly going through each signal parameters.